site-cookbooks/minecraft/attributes/default.rb

```diff
@@ -9,6 +9,7 @@
 
 default["minecraft"]["server"]["url"] = "https://launcher.mojang.com/v1/objects/3cf24a8694aca6267883b17d934efacc5e44440d/server.jar"
 default["minecraft"]["server"]["java"]["xmx"] = "4G"
+default["minecraft"]["server"]["java"]["g_xmx"] = "1G"
```

Comment:
man, you have been lazy here :D 

I'll change this to default["geyser"] later :D

Reply:
> man, you have been lazy here :D
> 
Not only lazy... It's wrong, since I didn't change the recipe that generates the systemd unit accordingly! :-/

> I'll change this to default["geyser"] later :D
>
Truth be told, I didn't see it! :-(

Let me see if I can put something together and send a new PR...

Reply:
Ok, @gbagnoli , you did it yourself already... Then I guess it's your fault if geyser doesn't start with:

```
Dec 10 05:15:46 whitestone systemd[1]: Started Geyser Server for Bedrock clients.
Dec 10 05:15:47 whitestone java[1030]: Invalid maximum heap size: -Xmx
Dec 10 05:15:47 whitestone java[1030]: Error: Could not create the Java Virtual Machine.
Dec 10 05:15:47 whitestone java[1030]: Error: A fatal exception has occurred. Program will exit.
```

In fact, the command line now is: `ExecStart=/usr/bin/java -Xmx -Xms512M -jar geyser.jar nogui`, and that's because the recipe still has this:

```
ExecStart=/usr/bin/java -Xmx#{jconf["g_xmx"]} -Xms#{jconf["xms"]} -jar geyser.jar nogui
```

Fixing...